### PR TITLE
update opcode.execute

### PR DESCRIPTION
**Pull request by sczembor**
Closes #x

## Proposed Changes

+ Updated `CHANGELOG`
+ Additional unit tests to test the returning value from the `execute` method of the opcodes.

### CODE REVIEW

**Review by vuvoth — APPROVED** [review]

File: packages/runtime/src/interpreter/opcode-list.ts [hunk 1]
```diff
@@ -615,6 +636,14 @@ export class Sha256 extends Op {
 		assertLen(args.length, 0, line);
 	}
 
+	computeCost(): number {
+		if (this.interpreter.tealVersion > 1) {
+			return 35;
+		} else {
+			return 7;
+		}
```
Comment (vuvoth) [inline packages/runtime/src/interpreter/opcode-list.ts:644]:
```suggestion
		return (this.interpreter.tealVersion > 1) ? 35 : 7;
```

File: packages/runtime/src/interpreter/opcode-list.ts [hunk 2]
```diff
@@ -649,6 +674,14 @@ export class Sha512_256 extends Op {
 		assertLen(args.length, 0, line);
 	}
 
+	computeCost(): number {
+		if (this.interpreter.tealVersion > 1) {
+			return 45;
+		} else {
+			return 9;
+		}
```
Comment (vuvoth) [inline packages/runtime/src/interpreter/opcode-list.ts:682]:
```suggestion
		return (this.interpreter.tealVersion > 1) ? 45 : 9;
```

**Review by vuvoth — APPROVED** [review]
LGTM!

File: packages/runtime/src/interpreter/opcode-list.ts [hunk 3]
```diff
@@ -684,6 +713,14 @@ export class Keccak256 extends Op {
 		assertLen(args.length, 0, line);
 	}
 
+	computeCost(): number {
+		if (this.interpreter.tealVersion > 1) {
```
Comment (rmeena840) [inline packages/runtime/src/interpreter/opcode-list.ts]: use ternary operator as suggested by @vuvoth

File: packages/runtime/src/interpreter/opcode-list.ts [hunk 4]
```diff
@@ -716,6 +749,10 @@ export class Ed25519verify extends Op {
 		assertLen(args.length, 0, line);
 	}
 
+	computeCost(): number {
+		return 1900;
```
Comment (rmeena840) [inline packages/runtime/src/interpreter/opcode-list.ts:741]: Would be better to store it in constants.

File: packages/runtime/src/interpreter/opcode-list.ts [hunk 5]
```diff
@@ -2328,8 +2443,10 @@ export class Int extends Op {
 		this.uint64 = uint64;
 	}
 
-	execute(stack: TEALStack): void {
+	execute(stack: TEALStack): number {
 		stack.push(this.uint64);
+
```
Comment (rmeena840) [inline packages/runtime/src/interpreter/opcode-list.ts:2436]: empty line???

File: packages/runtime/src/interpreter/opcode-list.ts [hunk 6]
```diff
@@ -2351,9 +2468,11 @@ export class Byte extends Op {
 		[this.str, this.encoding] = getEncoding(args, line);
 	}
 
-	execute(stack: TEALStack): void {
+	execute(stack: TEALStack): number {
 		const buffer = convertToBuffer(this.str, this.encoding);
 		stack.push(new Uint8Array(buffer));
+
```
Comment (rmeena840) [inline packages/runtime/src/interpreter/opcode-list.ts:2462]: Same

File: packages/runtime/src/interpreter/opcode-list.ts [hunk 7]
```diff
@@ -684,6 +705,14 @@ export class Keccak256 extends Op {
 		assertLen(args.length, 0, line);
 	}
 
+	computeCost(): number {
+		if (this.interpreter.tealVersion > 1) {
+			return 130;
+		} else {
+			return 26;
+		}
```
Comment (sczembor) [inline packages/runtime/src/interpreter/opcode-list.ts:713]:
```suggestion
return (this.interpreter.tealVersion > 1) ? 130 : 26;
```

File: packages/runtime/src/interpreter/opcode-list.ts [hunk 8]
```diff
@@ -2558,6 +2687,8 @@ export class SetBit extends Op {
 			}
 			stack.push(target);
 		}
+
```
Comment (rmeena840) [inline packages/runtime/src/interpreter/opcode-list.ts:2678]: try to remove empty line wherever possible.

File: packages/runtime/src/interpreter/opcode-list.ts [hunk 9]
```diff
@@ -3771,6 +3972,10 @@ export class EcdsaPkDecompress extends Op {
 		this.curveIndex = Number(args[0]);
 	}
 
+	computeCost(): number {
+		return 650;
```
Comment (rmeena840) [inline packages/runtime/src/interpreter/opcode-list.ts:3964]: again constant

**Review by rmeena840 — CHANGES_REQUESTED** [review]
Reviewed

File: packages/runtime/src/interpreter/opcode-list.ts [hunk 10]
```diff
@@ -637,11 +637,7 @@ export class Sha256 extends Op {
 	}
 
 	computeCost(): number {
-		if (this.interpreter.tealVersion > 1) {
-			return 35;
-		} else {
-			return 7;
-		}
+		return (this.interpreter.tealVersion > 1) ? 35 : 7;
```
Comment (rmeena840) [inline packages/runtime/src/interpreter/opcode-list.ts:640]: You can remove the brackets.

File: packages/runtime/src/interpreter/opcode-list.ts [hunk 11]
```diff
@@ -714,11 +706,7 @@ export class Keccak256 extends Op {
 	}
 
 	computeCost(): number {
-		if (this.interpreter.tealVersion > 1) {
-			return 130;
-		} else {
-			return 26;
-		}
+                return (this.interpreter.tealVersion > 1) ? 130 : 26;
```
Comment (rmeena840) [inline packages/runtime/src/interpreter/opcode-list.ts:709]: here too

**Review by rmeena840 — APPROVED** [review]

**Pull request merged**